layouts/_default/baseof.html

```diff
@@ -1,47 +0,0 @@
-<!DOCTYPE html>
-<html lang="{{ .Site.LanguageCode }}">
-  <head>
-    {{ partial "meta.html" . }}
-    <title>{{ block "title" . }}{{ .Site.Title }}{{ end }}</title>
-
-    <!-- Render theme css -->
-    {{ partial "css.html" . }}
-
-    <!-- Render custom header -->
-    {{ partial "head.html" . }}
-
-    <!-- Refer: https://regisphilibert.com/blog/2018/08/hugo-multilingual-part-1-managing-content-translation/ -->
-    {{ if .IsTranslated }}
-      {{ range .Translations }}
-      <link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .RelPermalink }}" title="{{ .Language.LanguageName }}" />
-      {{ end }}
-    {{ end }}
-    {{ template "_internal/twitter_cards.html" . }}
```

Comment:
Just checking: we don't need this anymore? It's unrelated to analytics, and IIRC this was a deliberate fix at some point.

Reply:
Looking into it. If needed, I will fix this in the theme.

Reply:
https://github.com/numpy/numpy.org/pull/345

Reply:
See #590 .